src/main/pages/che-7/overview/con_che-plug-ins.adoc

```diff
@@ -1,7 +1,7 @@
 [id="{prod-id-short}-plug-ins_{context}"]
-= {prod-short} plug-ins
+= `Che Plugin` plug-ins
```

Comment:
Again, I think this whole concept module doesn't talk about the `Che Plugin` plug-in type but about `{prod-short} plug-ins` in general.